avm/utilities/pipelines/e2eValidation/resourceRemoval/Initialize-DeploymentRemoval.ps1

```diff
@@ -95,6 +95,10 @@ function Initialize-DeploymentRemoval {
             'Microsoft.Resources/resourceGroups'
         )
 
+        $removeLastSequence = @(
```

Comment:
Just to double check: If multiple elements are specified in this list, in which order will they be removed?

Reply:
Exactly. You can see this tested here: 
https://github.com/Azure/bicep-registry-modules/blob/f1012df886a4052b67dd87e385b87c4c18f8ce62/avm/utilities/tests/pipelines/Get-OrderedResourcesList.tests.ps1#L77-L144

(different maybe PR that I mentioned)

Reply:
Hence I was so keen on having a Pester test 😄